images/hipstr/Dockerfile

```
@@ -0,0 +1,34 @@
+FROM ubuntu:20.04
+ARG DEBIAN_FRONTEND=noninteractive
+ARG VERSION=${VERSION:-v0.6.2}
+WORKDIR /app
+# hadolint ignore=DL3008
+RUN apt-get update -y && apt-get install -y --no-install-recommends -qqy \
+    apt-transport-https \ 
+    ca-certificates \
+    git \
+    g++ \
+    libbz2-dev \
+    libcurl4-openssl-dev \
+    libhts-dev \
+    liblzma-dev \
+    libssl-dev\
+    make \
+    zlib1g-dev \
+     && apt-get clean \
+     && rm -rf /var/lib/apt/lists/*
+
+RUN update-ca-certificates
+
+# Install HipSTR
+RUN git clone -b ${VERSION} https://github.com/HipSTR-Tool/HipSTR && \
+    cd HipSTR && \
+    make && \
+    cd ..
```

Can you merge these runs into one RUN, it saves on docker layers:

```suggestion
RUN apt-get update -y && apt-get install -y --no-install-recommends -qqy \
    apt-transport-https \ 
    ca-certificates \
    git \
    g++ \
    libbz2-dev \
    libcurl4-openssl-dev \
    libhts-dev \
    liblzma-dev \
    libssl-dev\
    make \
    zlib1g-dev \
    && apt-get clean \
    && rm -rf /var/lib/apt/lists/* \
    && update-ca-certificates \
    && git clone -b ${VERSION} https://github.com/HipSTR-Tool/HipSTR \
    && cd HipSTR && make && cd ..
```